nautobot/docs/additional-features/jobhooks.md

```diff
@@ -0,0 +1,21 @@
+# Jobhooks
+
+A jobhook is a mechanism for automatically starting a job when an object is changed. Jobhooks are similar to [webhooks](../models/extras/webhook.md) except change events initiate a `JobHookReceiver` job instead of a web request. Jobhooks are configured in the web UI under Jobs > Jobhooks.
+
+## Jobhook receivers
+
+Jobhooks are only able to initiate a specific type of job called a jobhook receiver. These are jobs that subclass the `nautobot.extras.jobs.JobHookReceiver` class. Jobhook receivers are similar to normal jobs except they are hard coded to accept only an `object_change` [variable](jobs.md#variables). Jobhook receivers are hidden from the UI by default but are otherwise identical to any other job.
```

Comment:
```suggestion
Jobhooks are only able to initiate a specific type of job called a jobhook receiver. These are jobs that subclass the `nautobot.extras.jobs.JobHookReceiver` class. Jobhook receivers are similar to normal jobs except they are hard coded to accept only an `object_change` [variable](jobs.md#variables). Jobhook receivers are hidden from the jobs listing UI by default but are otherwise identical to any other job.
```